content/en/docs/set-up/multisig.md

```diff
@@ -55,6 +55,21 @@ lotus msig approve walletAddress transactionID proposerAddress destinationAddres
 
 The value of `transactionID`, `proposerAddress`, `destinationAddress` and `value` must match the values used in the proposal.
 
+## Cancel a pending multisig proposal
+
+
+Use `lotus msig cancel` to cancel a pending multisig transaction. 
+
+```shell with-output
+lotus msig cancel walletAddress transactionID destinationAddress value
+
+```
+The value of `walletAddress`, `transactionID`, `destinationAddress` and `value` must match the values used in the proposal.
```

Comment:
````suggestion
```

The value of `walletAddress`, `transactionID`, `destinationAddress` and `value` must match the values used in the proposal.
````